.github/workflows/deploy.yml

```
@@ -94,6 +101,24 @@ jobs:
           file: "fly.toml"
           field: "app"
 
+      - name: Deploy PR
+        id: pr-deploy
+        uses: superfly/fly-pr-review-apps@1.0.0
+        if: ${{ github.event_name == 'pull_request' && github.event.label.name == 'staging' }}
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+          FLY_REGION: dfw
+          FLY_ORG: personal
+
+      - name: Comment URL on PR
+        uses: unsplash/comment-on-pr@master
```

Let's lock this action to its [latest tag](https://github.com/unsplash/comment-on-pr/releases/tag/v1.3.1) so that things don't break if that lib breaks on `master`.

```suggestion
        uses: unsplash/comment-on-pr@v1.3.1
```